lib/runner.js

```diff
@@ -49,11 +51,17 @@ app.on('ready', function() {
    */
 
   parent.on('browser-initialize', function(options) {
+    var userPreload = resolve('userPreloadTemplate.js');
```

Comment:
mind if we rename this to `user-preload-template.js`? not a huge deal, but i try to keep everything lowercase to avoid any issues case-sensitive filesystems.